components/src/preact/mutationComparison/mutation-comparison-table.tsx

```diff
@@ -14,30 +15,29 @@ export interface MutationsTableProps {
 }
 
 export const MutationComparisonTable: FunctionComponent<MutationsTableProps> = ({ data, proportionInterval }) => {
-    const getHeaders = () => {
-        return [
-            {
-                name: 'Mutation',
-                sort: {
-                    compare: (a: string, b: string) => {
-                        return sortSubstitutionsAndDeletions(a, b);
-                    },
+    const headers = [
+        {
+            name: 'Mutation',
+            sort: {
+                compare: (a: Substitution | Deletion, b: Substitution | Deletion) => {
+                    return sortSubstitutionsAndDeletions(a, b);
                 },
```

Comment:
A matter of taste, but this could be shortened:

```suggestion
                compare: sortSubstitutionsAndDeletions,
```